domain/storageprovisioning/state/base_test.go

```diff
@@ -254,6 +256,38 @@ VALUES (?, ?, ?, ?, 0, 100, 'rootfs')
 	return storageInstanceUUID
 }
 
+func (s *baseSuite) getStorageID(
+	c *tc.C, storageInstanceUUID domainstorage.StorageInstanceUUID,
+) string {
+	var storageID string
+	err := s.DB().QueryRowContext(
+		c.Context(), `
```

Comment:
In future, for test helpers invoking SQL, just use the non-context aware functions.